core/system/src/mpu/vmpu_freescale_k64_mem.c

```diff
@@ -112,11 +114,11 @@ void vmpu_mem_switch(uint8_t src_box, uint8_t dst_box)
     }
     /* Copy the ACLs into the MPU regions. */
     for (t = K64F_MPU_REGIONS_STATIC; t < u; t++, rgd++) {
-        word = MPU->WORD[t];
-        word[0] = rgd->word[0];
-        word[1] = rgd->word[1];
-        word[2] = rgd->word[2];
-        word[3] = 1;
+        region = (MPU_Region *) MPU->WORD[t];
+        region->STARTADDR   = rgd->start_addr;
```

Comment:
Do not align the `=` signs as it becomes a pain to keep them aligned later on when we change this code :)